lib/preprocessCss.js

```diff
@@ -128,6 +128,7 @@ module.exports = (app, callback) => {
 
             // enable inline source mapping in dev mode
             if (env === 'development') {
+              options.outFile = ''
```

Comment:
This was added as in the Sass documentation for [`sourceMap`](https://sass-lang.com/documentation/js-api#sourcemap), it states

> If this option is true, the path is assumed to be the outFile option with .map added to the end. If it’s true and the outFile option isn’t passed, it has no effect.

LibSass didn't seem to actually assert this statement but dart-sass does. Not sure if this is the best solution or not as `outFile` technically is not used in `preprocessCss` after the Sass / SCSS file is parsed and compiled down to CSS.

Reply:
Added the comment into the file upon your suggestion.